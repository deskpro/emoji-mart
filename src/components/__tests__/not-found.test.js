import React from 'react'
import NotFound from '../not-found'
import renderer from 'react-test-renderer'

import icons from '../../../data/icons.yml'
import categories from '../../../data/categories.yml'

const i18n = {
  notfound: 'No Emoji Found',
}

test('Renders <NotFound> component', () => {
  const emojiProps = {
    native: true,
    skin: 1,
    size: 24,
    set: 'apple',
    sheetSize: 64,
    forceSize: true,
    tooltip: false,
  }
  const component = renderer.create(
    <NotFound
      data={{ icons, categories }}
      notFound={() => {}}
      notFoundEmoji={'sleuth_or_spy'}
      emojiProps={emojiProps}
      i18n={i18n}
    />,
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
