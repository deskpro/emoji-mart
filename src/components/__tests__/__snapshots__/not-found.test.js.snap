// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <NotFound> component 1`] = `
<div
  className="emoji-mart-no-results"
>
  <div
    className="emoji-mart-no-results-label"
  >
    No Emoji Found
  </div>
</div>
`;
